cluster-autoscaler/FAQ.md

```diff
@@ -645,6 +645,25 @@ When using this class, Cluster Autoscaler performs following actions:
     * Adds a Provisioned=True condition to the ProvReq if the node group scale up request is successful.
     * Adds a BookingExpired=True condition when the 10-minute reservation period expires.
 
+### How can I tune Cluster Autoscaler's performance for processing ProvisioningRequests?
+
+Cluster Autoscaler can be run in batch processing mode for CheckCapacity ProvisioningRequests. In this mode, Cluster Autoscaler processes multiple CheckCapacity ProvisioningRequests in a single iteration. This mode is useful for scenarios where a large number of CheckCapacity ProvisioningRequests need to be processed.
+
+However, enabling batch processing for CheckCapacity ProvisioningRequests can adversely affect the performance of processing other types of ProvisioningRequests and incoming pods since iterations where CheckCapacity ProvisioningRequests are processed will take longer and scaleup for other types of ProvisioningRequests and incoming pods will not be processed during that time.
```

Comment:
```suggestion
However, enabling batch processing for CheckCapacity ProvisioningRequests can adversely
affect the performance of processing other types of ProvisioningRequests and incoming pods
since iterations where CheckCapacity ProvisioningRequests are processed will take longer
and scaleup for other types of ProvisioningRequests and incoming pods will not be processed
during that time.
```

Reply:
Small nit: Wrap the lines